src/lib/ai.ts

```diff
@@ -614,11 +623,14 @@ class WorkflowState<CTX> {
     private attempts = new Map<number, number>()
     private rewinder = 0
     private lastRewindTo = 0
+    public workflowRunId: string
```

Comment:
🤔  I'd rather have it private with a `getRunId()` method. As far as I can see it is not supposed to be set from anywhere, but it can...